### ISSUE TITLE
Exposure to propylthiouracil

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9992]
**Preferred term label:**
Exposure to propylthiouracil

**Synonyms**
PTU

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Telogen effluvium

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000093
- label: Propylthiouracil exposure
- parents: [Medication history (HP:4000112)](https://hpo.jax.org/app/browse/term/HP:4000112)
- pmid: 
- def: History of having been treated with propylthiouracil.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9992
- synonyms: Exposure to propylthiouracil(NONE)
